backend/tests/penetration_testing/invoice_pentest.py

```
@@ -0,0 +1,247 @@
+This script tests security aspects of the invoice API endpoints:
+1. File Upload Security Testing
+2. Invoice Access Control Testing
+3. Invoice Data Injection Testing
+
+API Endpoints:
+- Base URL: http://localhost:3000
+- Invoice API: http://localhost:3000/api/invoices
+"""
+
+import requests
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)